src/background/utils/tester.js

```diff
@@ -11,6 +11,10 @@ hookOptions((changes) => {
 });
 const RE_HTTP_OR_HTTPS = /^https?$/i;
 
+let bltCache;
```

Comment:
somehow I thought it was for BlackListTest but it's actually TestBlackList so now I'm not sure how to rename it because tbl is commonly used for tables...